coding/python.md

```diff
@@ -0,0 +1,20 @@
+---
+layout: base
+---
+
+# Python code standards
+
+This document outlines the rules for writing Python code across all our Projects.
+
+## Code style and formatting
+
+All Code should be written in accordance with [PEP 8](https://pep8.org/).
+To correctly format all your code please use [black](https://github.com/ambv/black). Since it differs in a small way from the standard **PEP8** make sure to pass `--line-length 79` when using it. The reason for black to do this is outlined [here](https://github.com/ambv/black#line-length).
+
+## Testing
+
+All logic should be tested. If you add something new to the Codebase make sure that you add sufficient testing to ensure longterm maintainability of our projects.
```

Comment:
Unfortunately I think this is just too far from either the reality of our existing codebases or our current practice to be taken seriously. Maybe we could say something like:

```suggestion
Although tests are lacking on many of our existing projects, writing tests is strongly encouraged. Please write tests for any new code if at all possible, and any efforts to improve test coverage on any project would be greatly appreciated.
```